autospec/specfiles.py

```diff
@@ -572,6 +572,50 @@ def write_license_files(self):
                 file2 = file.replace("/", "_")
                 self._write_strip("cp " + file + " %{buildroot}/usr/share/package-licenses/" + self.name + "/" + file2 + "\n")
 
+    def write_profile_payload(self, pattern=None):
+        """Write the profile_payload specified for this package."""
+        if not config.profile_payload:
+            return
+        if not config.profile_payload[0]:
+            return
```

Comment:
if not config.profile_payload or not config.profile_payload[0] probably but up to you.

Reply:
I will remove the second conditional, since we determined that it's not needed.